src/engine/blocks.js

```diff
@@ -298,7 +298,7 @@ class Blocks {
             if (!block.fields[args.name]) return;
             if (args.name === 'VARIABLE') {
                 // Get variable name using the id in args.value.
-                const variable = optRuntime.getEditingTarget().lookupVariableById(args.value);
```

Comment:
The only usage of changeBlock is line 217, and args.value passes in the name of the variable